java/src/jmri/jmrit/catalog/PreviewDialog.java

```diff
@@ -446,6 +432,8 @@ private long availableMemory() {
             for (int i = 0; i < memoryTest.size(); i++) {
                 memoryTest.remove(i);
             }
+            System.gc();
```

Comment:
Garbage Collection will have happened (at least in OpenJDK) before the OutOfMemoryError is thrown. Are you seeing a performance benefit from running it here?